### PR TITLE
Update yr provider to new api 

### LINKED ISSUE #3189: yr provider not working for weather - yr api deprecation
Using yr as the weather provider no longer works - it looks like it might be due to the sunrise 2.0 API being end of life.
Hitting the 2.0 URL that the yr provider uses returns an error: 
```
404 Not Found
The specified version number is end-of-lifed for this product
```
Their documentation confirms this occured on the 1st September 2023 - see the changelog at https://api.met.no/weatherapi/sunrise/3.0/documentation

### PR DESCRIPTION
**Pull request by martingron**
Some changes after yr api was deprecated and replaced with a new one.

Fixes #3189

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	"description": "The open source modular smart mirror platform.",
 	"main": "js/electron.js",
 	"scripts": {
-		"start": "DISPLAY=\"${DISPLAY:=:0}\" ./node_modules/.bin/electron js/electron.js",
-		"start:dev": "DISPLAY=\"${DISPLAY:=:0}\" ./node_modules/.bin/electron js/electron.js dev",
+		"start": ".\\node_modules\\.bin\\electron js\\electron.js",
```
Comment (rejas) [inline package.json:7]: Please revert these changes

**Review by rejas — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (martingron) [reply package.json]: Done :)

**Review by rejas — COMMENTED** [review]
Thx, will check once I am back at a pc later today's

**Comment by rejas** [comment]
One mall thing: run the linting task over the files and commit again.

**Comment by martingron** [comment]
linting task?

**Comment by rejas** [comment]
Npm run lint:prettier

**Review by rejas — CHANGES_REQUESTED** [review]
Please run `npm run lint:prettier` and commit the changes

**Review by rejas — APPROVED** [review]

**Pull request merged**